index.html

```diff
@@ -41,5 +41,142 @@ <h1 class="header-text">Hey there. I’m Raphael
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+        </div>
+
+
+        <section>
+            <div class="about-me">
+                <h3>About me</h3>
+                <div class="about-me-text">
+                 <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="my-resume"><p>Get My Resume</p></div>
+            </div>
+        </section>
+
+        <section>
+            <div class="skills-container">
+                <div class="skill-item">
+                    <img src="img/icon-languages.svg" alt="Language">
+                    <p>Language</p>
+                    <div class="skill-list"><div>JavScript</div><div>Ruby</div><div>html</div><div>CSS</div></div>
+                </div>
+
+                <div class="skill-item">
+                    <img src="img/icon-frameworks.svg" alt="Frameworks">
+                    <p>Frameworks</p>
+                    <div class="skill-list"><div>Bootstrap</div><div>Ruby on rails</div><div>RSpec</div><div>Capybara</div><div>Selenium</div></div>
+                </div>
+
+                <div class="skill-item">
+                    <img src="img/icon-skills (1).svg" alt="Skills">
+                    <p>Skills</p>
+                    <div class="skill-list"><div>Codekit</div><div>GitHub</div><div>Codepen</div><div>Gitlab</div><div>Terminal</div></div>
+                </div>
```

Comment:
Every other card on the client side are looking perfect, because of more language tags inside, the internal spacing on this one doesn't match the design. I suggest aligning contents in this card and others with `flex` to this card in order to make the UI consistent.

<img width="260" alt="card" src="https://user-images.githubusercontent.com/84930355/205119105-6e268fec-943c-41aa-93d2-0f8422aa3ed5.png">